### DISCUSSION TITLE
Is auth lib ready for production?

### DISCUSSION THREAD

**Discussion by Segamegadrive** [go-pkgz/auth#115] [General]
Hi there, I found this go-pkgz/auth lib for authentication. Is this lib ready for production?
I'm using gin-gonic as web framework to develop REST APIs and was looking for some genuine auth lib based on Golang.

Any advice or other libs you may know of would be greatly appreciated. 
Thank you

**Comment by umputun** [comment]
this library was born as an extraction of the code from a production system. As of now, it is stable enough to be used in the real world and we maintain semver and back compatibility promise.

**Reply by Segamegadrive** [reply]
Thank you so much for prompt response umputun. I really hope this lib grows more.
